app/assets/stylesheets/preact/article-form.scss

```diff
@@ -361,21 +361,19 @@
 }
 
 .articleform__body {
-  min-height: calc(80vh - 370px);
+  height: calc(80vh - 265px);
   width: 98%;
   border: 0;
   font-size: 15px;
   resize: none;
   padding: 20px 6px;
   font-family: var(--ff-accent);
-  // scrollbars not needed because the preact-textarea-autosize package
-  // automatically resizes the article form
-  overflow: hidden;
   background: var(--card-bg);
   color: var(--card-color);
+  overflow-y: auto;
   @media screen and (min-width: 600px) {
     font-size: 18px;
-    min-height: calc(85vh - 280px);
+    height: calc(85vh - 200px);
```

Comment:
Hi @aviskarkc10, it looks great! just a small suggestion here, for big screens:
 
```suggestion
    height: calc(82vh - 200px);
```

before:

![Screenshot_2020-05-14 New Post - DEV(local) Community 👩‍💻👨‍💻(1)](https://user-images.githubusercontent.com/146201/81917987-34d2ed80-95d6-11ea-8a19-3e0224506d3a.png)

after:

![Screenshot_2020-05-14 New Post - DEV(local) Community 👩‍💻👨‍💻](https://user-images.githubusercontent.com/146201/81917998-3997a180-95d6-11ea-884a-bd6cbbb639fe.png)

Reply:
@rhymes I have added the change